var/spack/repos/builtin/packages/py-azure-storage-blob/package.py

```diff
@@ -40,7 +48,7 @@ class PyAzureStorageBlob(PythonPackage):
     depends_on("py-typing-extensions@4.3:", type=("build", "run"), when="@12.17:")
     depends_on("py-typing-extensions@4.0.1:", type=("build", "run"), when="@12.15:")
     depends_on("py-isodate@0.6.1:", type=("build", "run"), when="@12.15:")
+    depends_on("py-msrest@0.7.1:", type=("build", "run"), when="@12.14")
+    depends_on("py-msrest@0.6.21:", type=("build", "run"), when="@:12.14")
```

Comment:
Both of these include 12.14, might be a mistake here (I realize the existing code already contained this mistake)

Reply:
Fixed